release/ray_release/reporter/db.py

```diff
@@ -1,18 +1,109 @@
 import time
+import re
 import json
 import boto3
+from typing import List
 from botocore.config import Config
 
 from ray_release.reporter.reporter import Reporter
 from ray_release.result import Result
 from ray_release.config import Test
 from ray_release.logger import logger
 
+TRACEBACK_PATTERN = "Traceback (most recent call last)"
+
 
 class DBReporter(Reporter):
     def __init__(self):
         self.firehose = boto3.client("firehose", config=Config(region_name="us-west-2"))
 
+    def compute_crash_pattern(self, logs: str) -> str:
+        stack_trace = self._compute_stack_trace(logs.splitlines())
+        return self._compute_signature(stack_trace)[:4000]  # limit of databrick field
```

Comment:
This doesn't really have anything to do with the Databricks reporter. Can we move the crash pattern parsing into a separate python file?